src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -674,16 +674,42 @@ class ContentSidebar extends PureComponent<Props, State> {
         });
     };
 
+    /**
+     * Adds a task to the tasks state and increases total_count.
+     *
+     * @param {Task} task - The newly created task from the API
+     * @return {void}
+     */
+    onTaskCreateSuccess(task: Task): void {
```

Comment:
Minor feedback regarding this file. In my PR I renamed some things to have more consistent naming to the other API functions. Something like `createTask` and `createTaskSuccessCallback`

Reply:
Done. What do you think we should do with the props with those names? IE) `onTaskCreate`, `onCommentCreate` etc.